01-node-tutorial/answers/writeWithPromisesAwait.js

```diff
@@ -0,0 +1,23 @@
+const { writeFile, readFile } = require('fs').promises;
+
+const writer = async () => {
+  try {
+    for (let i = 0; i < 3; i++) {
+      await writeFile('../content/temp.txt', `some text ${i}\n`, { flag: 'a' });
+    }
```

Comment:
This is nice optimization! However, be careful of using `async` within loops. They may or may not work because some loops and methods (like `.map` and `.filter`) can't be async!